containers/e2b-sandbox/e2b.toml

```diff
@@ -0,0 +1,14 @@
+# This is a config for E2B sandbox template.
```

Comment:
I don't see this file getting used anywhere?

Reply:
It's used by the E2B CLI, it's something like Cloudflare Workers have their [`wrangler.toml`](https://developers.cloudflare.com/workers/wrangler/configuration/)